### ISSUE TITLE
Ability to specify a version instead of the latest one

### ISSUE THREAD

**Issue by ustun** [johnwalker/lein-plz#23]
Should this tool allow specifying the version a la npm?

Kind of similar to `npm install foo@0.5.0 --save`?

**Comment by johnwalker** [comment]
Yeah, that's a great idea. We should also let people specify versions when they add groups of dependencies.

**Comment by roman01la** [comment]
Is anybody working on this? I'd like to implement this.
@johnwalker How do you think the command should look like?
for example: `lein plz add core.async@0.2.374`, is it good?

**Comment by johnwalker** [comment]
Yeah, that sounds awesome. No one is working on it to my knowledge.
